staging/publishing/rules.yaml

```diff
@@ -630,6 +630,8 @@ rules:
       dir: staging/src/k8s.io/kubectl
     name: master
     dependencies:
+      - repository: api
```

Comment:
This also needs to be added to release-1.15 rules below.

/hole

Reply:
/hold

Reply:
I see you've added that, all good?